datafusion/sqllogictest/test_files/topk.slt

```diff
@@ -370,7 +370,7 @@ query TT
 explain select number, letter, age, number as column4, letter as column5 from partial_sorted order by number desc, column4 desc, letter asc, column5 asc, age desc limit 3;
 ----
 physical_plan
-01)SortExec: TopK(fetch=3), expr=[number@0 DESC, column4@3 DESC, letter@1 ASC NULLS LAST, column5@4 ASC NULLS LAST, age@2 DESC], preserve_partitioning=[false], sort_prefix=[number@0 DESC, letter@1 ASC NULLS LAST]
+01)SortExec: TopK(fetch=3), expr=[number@0 DESC, letter@1 ASC NULLS LAST, age@2 DESC], preserve_partitioning=[false], sort_prefix=[number@0 DESC, letter@1 ASC NULLS LAST]
```

Comment:
this seems like a plan improvement 👍 as now there are fewer exprs in the sort expression (given the comments in the test, it seems like this was the expected results anyways)

Reply:
So wondering why the previous one doesn't do what the comment said.

Reply:
The old equivalence system was not performing normalizations in certain cases. Now, as we always store orderings in normal form (in addition to their original form), we don't miss these.